### PR TITLE
Add plugin/code highlight

### LINKED ISSUE #9: new plugin: code highlighting
A simple code highlighting plugin e.g. using https://highlightjs.org/

Maybe it's also possible to have an executable code plugin, e.g. with https://codeocean.com/ or https://tech.io/

### PR DESCRIPTION
**Pull request by FWirtz**
Impressions:
image
image

Closes #9

### CODE REVIEW

**Comment by FWirtz** [comment]
@Knorrke Finished from my side. Ready for review!

CC @janrenz @Trunken

File: plugins/content/highlight/src/Component/Input/index.js [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+class Input extends Component {
+    render() {
+        const style = {
+            textarea: {
+                height: '250px',
+                width: '100%',
+                padding: '5px',
+                fontFamily: 'Menlo, Monaco, "Courier New", monospace'
+            },
+            submit: {
```
Comment (Knorrke) [inline plugins/content/highlight/src/Component/Input/index.js]: I guess this styling isn't needed anymore, since the button was removed?

Comment (FWirtz) [reply plugins/content/highlight/src/Component/Input/index.js]: totally right!

**Comment by FWirtz** [comment]
concerning:
> Also, if there's an easy fix: **Pressing tab key in the inputarea switches the focus to the language input field**. Personally I'd expect it to indent the line by 4 spaces or something like that. Though this is open for debate, so it doesn't need to be included in the PR, if you don't think it's necessary!

The same applies for using the up/down arrow, they also result in leaving the focus of the window.

Now, I don't know how easy that is to fix, but here is why I didn't invest more time in that: **I don't think it's practical to write actual code inside the editor.** I would always **copy it from somewhere** else and be happy to leave the window quickly.

That might be personal preference though.

Should we capture the users focus `onEnter`, so that pressing "Tab", "Up/Down" etc. doesn't make the user leave the window?
What do you guys @Knorrke @janrenz think about that?

File: plugins/content/highlight/src/Component/Input/index.js (hunk 1 above)
Comment (FWirtz) [reply plugins/content/highlight/src/Component/Input/index.js]: will remove that

**Comment by Knorrke** [comment]
I think i'd favour capturing the focus. Especially for small sized code snippets I probably wouldn't start up an external programm... fast switching between the input fields seems not that important to mee. But I understand your point too, so we can also leave the code as it is and create a seperate issue for this discussion tagged as 'question'.

**Comment by FWirtz** [comment]
@Knorrke Agreed.
Ready to merge then 😊

**Pull request merged**